stories/solid/Virtualizer.stories.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * @jsxImportSource solid-js
+ */
+import type { Meta, StoryObj } from "storybook-solidjs";
+import { Virtualizer } from "../../src/solid";
+
+export default {
+  component: Virtualizer,
+} as Meta;
+
+const createRows = (num: number) => {
+  return Array.from({ length: num }).map((_, i) => i);
+};
+
+const COLUMN_WIDTHS = [100, 200, 300, 100, 200, 300, 100, 300, 400, 200];
+
+export const TableElement: StoryObj = {
```

Comment:
This is super basic but demonstrates it is possible to pass in elements for as and item. I can add more examples if you think it is appropriate